pkg/daemon/update.go

```diff
@@ -233,6 +243,26 @@ func (dn *Daemon) reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) *stri
 	return nil
 }
 
+func verifyUserFields(pwdUser ignv2_2types.PasswdUser) string {
```

Comment:
If you end up making changes to this PR please add a doc to this function. While it's not required being an internal function, it's still helpful for future developers.

Reply:
Done.